docs/user_guide/config.md

```diff
@@ -86,32 +86,39 @@ Below are all the possible Neurobagel environment variables that can be set in `
 
 ### Change security relevant variables
 
-For security purposes, we recommend the following additional steps for configuring your node if you are using a deployment profile that includes a graph store:
+For security and best practice purposes, follow the below additional steps to configure your node if you are using a deployment profile that includes a graph store:
 
-1. Review and change the values of the following variables in `.env`:
+1. **Change the values** of the following variables in `.env` from their defaults:
     - `NB_GRAPH_USERNAME`
-    - `NB_GRAPH_SECRETS_PATH`
     - `NB_GRAPH_DB`
-    - `NB_RETURN_AGG`
 
-2. Change the default passwords for the `admin` superuser and newly created graph database user (`NB_GRAPH_USERNAME`) for your graph store. 
-These are set to the contents of files called `NB_GRAPH_ADMIN_PASSWORD.txt` and `NB_GRAPH_PASSWORD.txt`, respectively, which are found by default in [`./secrets`](https://github.com/neurobagel/recipes/tree/main/secrets) in the Neurobagel recipes repo.
-To change the location of your password files, simply edit the variable `NB_GRAPH_SECRETS_PATH` in `.env` to point to a more secure directory where you have stored the two text files.
+2. **Replace the default passwords** for the `admin` superuser and the newly created graph database user (`NB_GRAPH_USERNAME`) for your graph store with your own secure passwords. 
 
-    Make sure to use secure passwords for `NB_GRAPH_ADMIN_PASSWORD.txt` and `NB_GRAPH_PASSWORD.txt`.
+    - These passwords are set using the contents of files named `NB_GRAPH_ADMIN_PASSWORD.txt` and `NB_GRAPH_PASSWORD.txt`, respectively. 
+    The default location of these files is [`./secrets`](https://github.com/neurobagel/recipes/tree/main/secrets) in the Neurobagel [`recipes`](https://github.com/neurobagel/recipes) repo.
+    - To generate a random password in the terminal, you can use:
+      ```bash
+      openssl rand -hex 16
+      ```
 
-    To generate a random password in the terminal, you can use:
+    - (Optional) Change the location of your password files to a more secure directory using the variable `NB_GRAPH_SECRETS_PATH`.
 
-    ```bash
-    openssl rand -hex 16
-    ```
+    ??? info "Graph store passwords are not meant for use by node query users"
+        The passwords specified in the deployment recipe are only used internally by the scripts that (automatically) set up and update the graph store, or to interact directly with the graph store (e.g., to modify database configuration or data).
```

Comment:
```suggestion
    ??? info "Graph store passwords are not meant for use by node query users"
        The passwords specified in the deployment recipe are only used internally by the scripts that (automatically) set up and update the graph store, 
        or to interact directly with the graph store (e.g., to modify database configuration or data).
        The passwords are also used to secure internal communication between your graph and its node API.
        An external query user is not able to query your graph directly.
```
Not sure this is clear yet. I think this could be a good clarification for folks confused by what the password does - but the phrasing is a bit odd. Maybe: "Graph store passwords are for internal communication only".